plugin/completion/flags_manager.py

```diff
@@ -190,10 +205,33 @@ def get_flags(self, separate_includes):
         if self._clang_complete_file.was_modified():
             log.debug(" .clang_complete modified. Load new flags.")
             self._flags = FlagsManager.flags_from_clang_file(
-                self._clang_complete_file, separate_includes)
+                self._clang_complete_file)
 
         # the flags are now in final state, we can return them
-        return self._flags
+        return FlagsManager.merge_flags(self._initial_flags, list(self._flags))
```

Comment:
I don't think there will be any duplicates as both parts are overwritten in flags manager and they are always merged anew.